to_copy/usr/bin/pep-filter.sh

```diff
@@ -0,0 +1,268 @@
+#!/bin/bash
+
+BASE_DIR=$(cd $(dirname "$0") && pwd -P)
+SCRIPT_NAME=$(basename "$0")
+
+FILTERS_DIR="/etc/encryptme/filters"
+DOMAIN_RE="^([A-Za-z0-9-]+\.)+[A-Za-z]{2,}"
+CIDR_RE="([0-9]{1,3}\.){3}[0-9]{1,3}(\/[0-9]{1,3})?"
+TMP_DIR="/tmp/$SCRIPT_NAME.$$" && mkdir -p "$TMP_DIR" \
+    || fail "Failed to create temporary directory '$TMP_DIR'"
+
+
+# JFK NOTES:
+# - ensure domains/IPs persist after reboot, container restart, etc
+
+usage() {
+    cat << EOF
+usage: $SCRIPT_NAME ACTION ARGS
+
+Automated DNS and IPv4 CIDR filtering based on arbitrary lists. Reads STDIN
+for a list of domains or IPv4 CIDR ranges.
+
+  - Domains are sync'd in $FILTERS_DIR and read by the DNS filter socket server
+  - IP sets are created and used in iptables to filter out IPs; writes rules to:
+     - /etc/iptables.save
```

Comment:
remove these references and instead make reference that this script must be used on container startup (e.g. server rebooted) to dynamically restore filtering rules

take care to not create dupes; better to reset+redo if needed

Reply:
Reference has been changed